torch/nn/quantized/dynamic/modules/rnn.py

```diff
@@ -308,30 +311,24 @@ def process_weights(ihhh, layer, suffix, dtype):
                         # weights and pack parameters in this order:
                         #
                         #   w_ih, w_hh
-                        weight_observer(weight)
-                        wt_scale, wt_zp = weight_observer.calculate_qparams()
-                        qweight = torch.quantize_per_tensor(
-                            weight.float(), float(wt_scale), int(wt_zp), torch.qint8)
-                        packed_weight = \
-                            torch.ops.quantized.linear_prepack(qweight, bias)
+                        qweight = _quantize_weight(weight, weight_observer)
```

Comment:
Need to observe weights prior to calling quantize_weight
``` python
weight_observer(weight)
qweight = _quantize_weight(weight, weight_observer)
```

Reply:
We also need tests for these changes

Reply:
I agree -- I will send tests as well